src/components/typography/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
+import { Slot } from "@radix-ui/react-slot"
+import { cva, type VariantProps } from "class-variance-authority"
+import { cn } from "../../lib/utils"
+
+export const typographyVariants = cva("", {
+  variants: {
+    variant: {
+        h1: "scroll-m-20 text-4xl font-semibold tracking-[-0.04em] lg:text-5xl",
+        h2: "scroll-m-20 border-b pb-2 text-3xl font-semibold tracking-[-0.04em] first:mt-0",
```

Comment:
All headers should follow the same convention. This one has custom border, padding, margin. We will end up with overriding this always in apps